app/views/email_authentications/invitations/new.html.erb

```diff
@@ -9,6 +9,9 @@
       <%= form.text_field :full_name, id: :full_name, required: true, autocomplete: "off", label: "Full name *", placeholder: "Full name" %>
       <%= form.text_field :email, id: :email, required: true, label: "Email *", placeholder: "Email" %>
       <%= form.text_field :role, id: :role, required: true, help: "CVHO, STS, State Official etc.", label: 'Job title *', placeholder: "Job title" %>
+      <div class="mt-3 mb-3">
+        <%= form.check_box :receive_approval_notifications, checked: true, value: "1", label: "Receive approval notifications" %>
```

Comment:
This helper can take a `checked` value and an `unchecked` value like in https://github.com/simpledotorg/simple-server/blob/4a3148e37bf12b615635ec9cae0ab547bf3febb5/app/views/admin/facility_groups/_form.html.erb#L83

Don't need the `== "1"` check if we do this.

Reply:
I just discovered that `true, false`, is actually incorrect 😓  I think we need to specify `"true", "false"` instead. The facility group form operates on Javascript so this isn't an issue, but the Rails form doesn't submit the unchecked value if we set it to `false`. Check out why [here](https://github.com/rails/rails/blob/44a9aedd7b8d65517b15bbbb7729f3f16991e23f/actionpack/lib/action_view/helpers/form_helper.rb#L1100).

In any case, I'll give `"true", "false"` a whirl!

Reply:
It works great 🚀

Reply:
Actually, we don't need to set values at all! "0" and "1" are designed to be assigned to boolean AR models already 🤦  #magic